pages/teleport.vue

```diff
@@ -27,6 +27,12 @@
         </div>
       </b-field>
 
+      <b-field>
+        <div class="is-flex is-align-items-center">
+          {{ routeMessage }}
+        </div>
+      </b-field>
```

Comment:
Well it's not really a field, I would use `<p class="subtitle">` instead plus `is-flex is-align-items-center` is not needed

Reply:
Thanks for the hack Leo!